src/main/java/com/google/cloud/tools/managedcloudsdk/install/SdkInstaller.java

```diff
@@ -173,11 +173,44 @@ public static SdkInstaller newInstaller(
       String userAgentString,
       boolean usageReporting,
       Map<String, String> environmentVariables) {
+    return SdkInstaller.newInstaller(
+        managedSdkDirectory,
+        version,
+        osInfo,
+        userAgentString,
+        usageReporting,
+        null,
+        environmentVariables);
+  }
+
+  /**
+   * Configure and create a new Installer instance.
+   *
+   * @param managedSdkDirectory home directory of google cloud java managed cloud SDKs
```

Comment:
what is "google cloud java" here?

Reply:
hmm i'm not sure what this refers to, this description was copy-pasted from another method, didn't look too closely at it. i guess it's google cloud sdks that are managed a java library? idk though, happy to remove

Reply:
Best to avoid copy pasta. Rewrite and perhaps rename the parameter to match what's happening in this method.